sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1136,7 +1150,23 @@ object Radar {
                 if (status == RadarStatus.SUCCESS) {
                     RadarSettings.setTripOptions(context, options)
 
+                    // store previous tracking options for post-trip
+                    // if tracking was false, previousTrackingOptions will be null
+                    val isTracking = Radar.isTracking()
+                    if (isTracking) {
+                        val previousTrackingOptions = Radar.getTrackingOptions()
+                        RadarSettings.setPreviousTrackingOptions(context, previousTrackingOptions)
+                    } else {
+                        RadarSettings.removePreviousTrackingOptions(context)
+                    }
+
+                    // if trackingOptions provided, startTracking
+                    if (trackingOptions != null) {
+                        Radar.startTracking(trackingOptions);
+                    }
+
                     // flush location update to generate events
+                    // TODO: still needed?
```

Comment:
@corypisano The goal here is to trigger a location update to be attached to the trip as soon as possible (so the trip has a location, has an ETA, etc.). `startTracking()` calls `updateTracking()`, which applies the desired intervals and will trigger a location update to the server in most cases (an exception might be if the last location update happened within `desiredSyncInterval`). Suggest we form an opinion on this, test it, and remove the `TODO` before we ship.

Reply:
If we're unsure, it's relatively harmless to keep it.